src/common/components/delegated-vesting/index.tsx

```diff
@@ -24,149 +33,147 @@ import parseAsset from "../../helper/parse-asset";
-
-                    return parseAsset(b).amount - parseAsset(a).amount;
-                },
-                formatter: (cell: any, row: DelegatedVestingShare) => {
-                    const vestingShares = parseAsset(row.vesting_shares).amount;
-
-                    return (
-                        <>
-                            {formattedNumber(vestsToSp(vestingShares, hivePerMVests), {suffix: "HP"})} <br/>
-                            <small>{row.vesting_shares}</small>
-                        </>
-                    );
-                },
-            },
-        ];
-
-        const pagination = {
-            sizePerPage: 8,
-            hideSizePerPage: true,
-        };
-
-        const tableProps = {
-            bordered: false,
-            keyField: "delegatee",
-            data,
-            columns,
-            pagination: data.length > pagination.sizePerPage ? paginationFactory(pagination) : undefined,
-        };
-
-        // @ts-ignore this is about the library's defaultSorted typing issue
-        const table = <BootstrapTable {...tableProps} />;
-
         return (
-            <div className="delegated-vesting-content">
-                <div className="table-responsive">{table}</div>
+            <div className={_c(`delegated-vesting-content ${inProgress ? "in-progress" : ""} ${hideList ? "hidden" : ""}`)}>
+                <div className="user-list">
+                    <div className="list-body">
+                        {data.length === 0 && <div className="empty-list">{_t("g.empty-list")}</div>}
+                        {data.map(x => {
+                            const vestingShares = parseAsset(x.vesting_shares).amount;
+                            const {delegatee: username} = x;
+
+                            const deleteBtn = (activeUser && activeUser.username === account.name) ? KeyOrHotDialog({
+                                ...this.props,
+                                activeUser: activeUser,
+                                children: <a href="#" className="undelegate">{_t("delegated-vesting.undelegate")}</a>,
+                                onToggle: () => {
+                                    const {hideList} = this.state;
+                                    this.stateSet({hideList: !hideList});
+                                },
+                                onKey: (key) => {
+                                    this.stateSet({inProgress: true});
+                                    delegateVestingShares(activeUser.username, key, username, "0.000000 VESTS")
+                                        .then(() => this.fetch())
+                                        .catch(err => error(formatError(err)))
+                                        .finally(() => this.stateSet({inProgress: false}))
+                                },
+                                onHot: () => {
+                                    delegateVestingSharesHot(activeUser.username, username, "0.000000");
```

Comment:
I believe `vesting_shares` should be in `0.000000 VESTS`, just numbers wouldn't work.